### PR TITLE
Fixes for docs

### PR DESCRIPTION
**Pull request by EricFromCanada**
## Description

Add a missing line break and specify that `rulesets` requires an array.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (non-breaking change that does not add functionality but updates documentation)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/markdownlint/markdownlint/blob/master/CONTRIBUTING.md) document.
- [x] Wrote [good commit messages](https://chris.beams.io/posts/git-commit/)
- [x] Feature branch is up-to-date with `master`, if not - rebase it
- [x] Added tests for all new/changed functionality, including tests for positive and negative scenarios
- [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences

### CODE REVIEW

**Review by jaymzh — APPROVED** [review]
Nice, thanks! Assuming tests pass, I'll merge.

**Comment by jaymzh** [comment]
Looks like a new version of rubocop is catching some ruby style things. I can fix that in a PR later today probably and then merge this. Or if you prefer you can send a PR.

File: mdl.gemspec [hunk 1]
```diff
@@ -11,6 +11,7 @@ Gem::Specification.new do |spec|
   spec.description = 'Style checker/lint tool for markdown files'
   spec.homepage = 'http://github.com/markdownlint/markdownlint'
   spec.license = 'MIT'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```
Comment (jaymzh) [inline mdl.gemspec:14]: should this be a string, or a bool?

Comment (EricFromCanada) [reply mdl.gemspec:14]: I copied it verbatim [from here](https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecrequiremfa).

Comment (jaymzh) [reply mdl.gemspec:14]: Nice. Just checking, wasn't sure.

**Pull request merged**